### PR TITLE
audiobooks

### PR DESCRIPTION
**Pull request by kiahjh**

### CODE REVIEW

File: apps/next-evans/lib/dates.ts [hunk 1]
```diff
@@ -45,3 +47,18 @@ export function newestFirst<
   }
   throw new Error(`unreachable`);
 }
+
+// hh:mm:ss
+export function formatDuration(durationSeconds: number): string {
```
Comment (jaredh159) [inline apps/next-evans/lib/dates.ts:52]: this formatting used to be done on the API, and i think you might have missed some edge cases, like if it's less than an hour, we don't want a leading `0:` or `:`. here are the test tuples from the swift unit tests, can you add a TS unit test to check all of these?

```ts
let cases = [
  [185, `3:05`],
  [11105, `3:05:05`],
  [19183, `5:19:43`],
  [3318, `55:18`],
  [3203, `53:23`],
  [9720, `2:42:00`],
  [3, `3`],
]
```

use the `test.each()` style, rather than manually iterating over the cases.

File: apps/next-evans/pages/audiobooks.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          <>
+            Explora los {sortedBooks.length} audiolibros a continuación u{` `}
+            <Link href={t`/audio-help`} className="subtle-link">
+              obtén ayuda aquí para escuchar
+            </Link>
+            .
+          </>
+        </Dual.P>
+      </div>
+      <div className="flex gap-x-8 gap-y-36 flex-wrap justify-center py-24 px-4 sm:px-20">
```
Comment (jaredh159) [inline apps/next-evans/pages/audiobooks.tsx:87]: two things here:

1. at `xl` the current production site shows **3** columns of audio books, but you're only showing 2. was this intentional?
2. check the deploy previews, the text of the buttons on this component is not correct.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
looks solid 👍

just a couple requested changes.

**Review by jaredh159 — APPROVED** [review]
